tests/integration/hpe_3par_manager.py

```diff
@@ -111,6 +111,7 @@ def hpe_inspect_volume(self, volume, **kwargs):
 
         volume_details = ['size', 'provisioning', 'flash_cache', 'compression', 'mountConflictDelay', 'importVol', 'cpg', 'snapcpg']
         qos_detail = ['enabled', 'maxIOPS', 'minIOPS', 'priority', 'vvset_name']
+        snapshots = inspect_volume['Status']['Snapshots'][0]

```

Comment:
This will throw keyerror with other tests

Reply:
added checks specific to spashots